src/components/HeroSection.vue

```diff
@@ -1,36 +1,31 @@
 <template>
   <div class="hero">
-    <img class="logo" alt="Logo" src="../assets/cc-logo_large_black.png">
+    <h1>Search for open content</h1>
```

Comment:
please update this to the latest version of the text proposed by @annatuma and @sarahpearson